# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../vdr-image/ffmpeg-cpp.patch
# Copyright (C) 2007 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- ./liboutput/encode.h.orig	2006-01-11 20:23:30.000000000 +0100
+++ ./liboutput/encode.h	2007-10-29 12:02:52.000000000 +0100
@@ -25,12 +25,16 @@
 #ifndef _ENCODE_H
 #define _ENCODE_H
 
+extern "C" {
+
 #ifdef FFMDIR
 #include <avcodec.h>
 #else
 #include <ffmpeg/avcodec.h>
 #endif 
 
+}
+
 #include "../setup-image.h"
 //#define TESTCODE
 
